src/integration.test.ts

```diff
@@ -297,6 +298,43 @@ describe(`Integration test for portal '${portal}'`, () => {
       expect(returnedEntry.data).toEqualUint8Array(dataLinkBytes);
     });
 
+    it("should set and get entry data", async () => {
+      const { publicKey, privateKey } = genKeyPairAndSeed();
+      const data = new Uint8Array([1, 2, 3]);
+
+      // Set the entry data.
+      await client.db.setEntryData(privateKey, dataKey, data);
+
+      // Get the entry data.
+      const { data: returnedData } = await client.db.getEntryData(publicKey, dataKey);
+
+      // Assert the returned data equals the original data.
+      expect(returnedData).toEqualUint8Array(data);
+    });
+
+    it("should set and delete entry data", async () => {
+      const { publicKey, privateKey } = genKeyPairAndSeed();
+      const data = new Uint8Array([1, 2, 3]);
+
+      // Set the entry data.
+      await client.db.setEntryData(privateKey, dataKey, data);
+
+      // Delete the entry data.
+      await client.db.deleteEntryData(privateKey, dataKey);
+
+      // Trying to get the deleted data should result in null.
+      const { data: returnedData } = await client.db.getEntryData(publicKey, dataKey);
+      // TODO: Should this equal null?
+      expect(returnedData).toEqualUint8Array(new Uint8Array(RAW_SKYLINK_SIZE));
+    });
+
+    it("should not be able to delete inexistent data", async () => {
```

Comment:
where is the assertion?

Reply:
I just reworked this whole test, sorry for the oversight.